recipe/meta.yaml

```diff
@@ -1,36 +1,47 @@
 {% set name = "pypng" %}
 {% set version = "0.20220715.0" %}
+{% set sha256 = "739c433ba96f078315de54c0db975aee537cbc3e1d0ae4ed9aab0ca1e427e2c1" %}
```

Comment:
As `sha256` is used only once then it would be better to remove this jinja2 variable. Profit: 1 line instead of 2 in the recipe

Reply:
Done.